kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -1996,38 +2093,83 @@ def cancel_sharedblock_self_fencer(self, req):
+        if not linux.wait_callback_success(_wait, timeout=retry_times*sleep_times):
+            raise Exception("report self fencer to mn failed for ps list %s" % ps_uuids)
+
     def do_heartbeat_on_sharedblock(self, cmd):
 
-        def _do_fencer_vg(vg, failure):
+        def _pause_vm_on_vg(vg):
+            for vm in lvm.get_running_vm_root_volume_on_vg(vg):
+                try:
+                    vm = get_vm_by_uuid(vm.uuid)
+                    vm.pause()
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)